app.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "DUB registry (mirror)",
+  "description": "A mirror of code.dlang.org",
+  "repository": "https://github.com/heroku/node-js-sample",
+  "logo": "https://dlang.org/images/dlogo_opengraph.png",
+  "keywords": ["d", "dlang", "dub", "mirror"]
+}
```

Comment:
Setting up a mirror will be only two clicks away - see e.g. https://devcenter.heroku.com/articles/heroku-button

A mongoDB addon is needed though, but there are many free ones (e.g. [mLab](https://elements.heroku.com/addons/mongolab)).